CMakeLists.txt

```diff
@@ -73,6 +74,7 @@ endif()
 # Tests
 #============================================================================
 if(WITH_TESTS)
+	find_program(PYTHON python REQUIRED)
```

Comment:
The [`REQUIRED` keyword for `find_program` is introduced in CMake 3.18](https://cmake.org/cmake/help/latest/command/find_program.html), but the minimum required CMake version is 3.1.

Reply:
Thanks for noticing.  Which do you prefer, increasing minimum cmake version or removing the REQUIRED keyword?

Reply:
I removed the REQUIRED keyword; let's keep some backwards compatibility.